core/src/components/input/test/shape/input.e2e.ts

```diff
@@ -0,0 +1,111 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('input: shape'), () => {
+    /**
+     * Solid fill is only available in MD theme.
+     */
+    test.describe('fill solid', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-input
+            shape="round"
+            fill="solid"
+            label="Email"
+            value="hi@ionic.io"
+            helper-text="Enter your email"
+            maxlength="20"
+            counter="true"
+          ></ion-input>
+        `,
+          config
+        );
+
+        const input = page.locator('ion-input');
+        await expect(input).toHaveScreenshot(screenshot(`input-shape-round-fill-solid`));
+      });
+      test('border radius should be customizable', async ({ page }) => {
+        await page.setContent(
+          `
+          <style>
+            ion-input {
+              --border-radius: 10px !important;
+            }
+          </style>
+
+          <ion-input
+            shape="round"
+            fill="solid"
+            label="Email"
+            label-placement="floating"
+            value="hi@ionic.io"
+            helper-text="Enter your email"
+            maxlength="20"
+            counter="true"
+          ></ion-input>
+        `,
+          config
+        );
+
+        const input = page.locator('ion-input');
+        await expect(input).toHaveScreenshot(screenshot(`input-fill-shaped-solid-custom`));
```

Comment:
```suggestion
        await expect(input).toHaveScreenshot(screenshot(`input-shape-round-fill-solid-custom`));
```

Can we update this to match the other naming and rename the screenshots / remove the old ones?

Reply:
https://github.com/ionic-team/ionic-framework/pull/29469/commits/1885f5e4d82fac3187fc9e654658c7aa035ed014